packages/hash/api/src/graphql/resolvers/middlewares/canAccessAccount.ts

```diff
@@ -11,7 +11,7 @@ import { ResolverMiddleware } from "./middlewareTypes";
 export const canAccessAccount: ResolverMiddleware<
```

Comment:
We are not yet using this middleware function with any of the re-implemented GQL mutations/queries, but have updated it here in case we do want to make use of it.